radio/src/gui/Taranis/menu_general_sdmanager.cpp

```diff
@@ -172,19 +172,31 @@ void onSdManagerMenu(const char *result)
   }
   else if (result == STR_PASTE) {
     f_getcwd(lfn, _MAX_LFN);
-    POPUP_WARNING(fileCopy(clipboard.data.sd.filename, clipboard.data.sd.directory, lfn));
-    REFRESH_FILES();
+    // if destination is dir, copy into that dir
+    if (!line[SD_SCREEN_FILE_LENGTH+1]) {
```

Comment:
Now I read this, I wonder why I didn't write a macro for this, it's difficult to read ...